src/mode/_test/highlight_rules_test.js

```diff
@@ -259,7 +259,8 @@ function testMode(modeName, i) {
     var Mode = require("../" + modeName).Mode;
     var tokenizer = new Mode().getTokenizer();
 
-    checkBacktracking(tokenizer);
+    // TODO this is too slow to run in regular tests
+    // checkBacktracking(tokenizer);
```

Comment:
`checkBacktracking` function implementation did not change - why do we need to comment it out? `checkBacktrackingWithRecheck` is what seems to be slow based on your comments, and it's not used anywhere, so we don't need to comment out `checkBacktracking` as well.

Reply:
checkBacktracking was my naive attempt to implement something similar to recheck, when it was not available yet. It had too many false positives and was needlessly cluttering test output with warnings that no one was reading. We probably should remove it entirely.